iati/resources/lib_data/validation_err_codes.yaml

```diff
@@ -0,0 +1,358 @@
+# A YAML file containing the error codes that may be output from validation.
+#
+# The structure of a code is:
+#
+#   - name-of-the-error-or-warning:
```

Comment:
Could add back ticks to the key names. This should signify that they are the keys and what follows is a definition.

Reply:
IMO, that'd confuse things because it'd look like you're supposed to include the backticks.

If doing anything along these lines, I'd look to make it clear what the descriptions are (since these are the things to be culled). I'm not, however, sure if there's a nice way to do this.

Reply:
Hrmm, not sure either way is perfect so let's not make any further changes at the moment.